src/mono/wasm/runtime/dotnet.d.ts

```diff
@@ -45,8 +45,8 @@ declare interface EmscriptenModule {
     HEAPF64: Float64Array;
     _malloc(size: number): VoidPtr;
     _free(ptr: VoidPtr): void;
-    print(message: string): void;
-    printErr(message: string): void;
+    out(message: string): void;
```

Comment:
This is API breaking change.

Reply:
Do we consider this API stable?

We can still get the deprecated ones with `-sINCOMING_MODULE_JS_API=print,printErr` if needed.

Reply:
> Do we consider this API stable?

No

> We can still get the deprecated ones with `-sINCOMING_MODULE_JS_API=print,printErr` if needed.

Yes please, I think these 2 are widely enough used even by Uno.

Reply:
They could be removed from .d.ts but should keep working.

Reply:
I'm not sure we want to be in the business of maintaining unstable emscripten apis that emscripten has dropped long term but we should document the change and short term compat is fine.